File: branches/Andean/Fantasy.Base/ServiceModel/AddressSetting.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Xml.Serialization;

namespace Fantasy.ServiceModel
{
    [XmlRoot("address")]
    public class AddressSetting
    {
        public AddressSetting()
        {
            Port = -1;
        }

        [XmlAttribute("host")]
        public string Host { get; set; }

        [XmlAttribute("port")]
        public int Port { get; set; }

        [XmlArray("contracts"),
        XmlArrayItem("contract", Type = typeof(string))]
        public string[] Contracts { get; set; }
    }
}

File: Fantasy.Base/src/fantasy/servicemodel/AddressSetting.java
package fantasy.servicemodel;

//C# TO JAVA CONVERTER TODO TASK: Java annotations will not correspond to .NET attributes:
//[XmlRoot("address")]
public class AddressSetting
{
	public AddressSetting()
	{
		setPort(-1);
	}

//C# TO JAVA CONVERTER TODO TASK: Java annotations will not correspond to .NET attributes:
	//[XmlAttribute("host")]
	private String privateHost;
	public final String getHost()
	{
		return privateHost;
	}
	public final void setHost(String value)
	{
		privateHost = value;
	}

//C# TO JAVA CONVERTER TODO TASK: Java annotations will not correspond to .NET attributes:
	//[XmlAttribute("port")]
	private int privatePort;
	public final int getPort()
	{
		return privatePort;
	}
	public final void setPort(int value)
	{
		privatePort = value;
	}

//C# TO JAVA CONVERTER TODO TASK: Java annotations will not correspond to .NET attributes:
	//[XmlArray("contracts"), XmlArrayItem("contract", java.lang.Class = typeof(String))]
	private String[] privateContracts;
	public final String[] getContracts()
	{
		return privateContracts;
	}
	public final void setContracts(String[] value)
	{
		privateContracts = value;
	}
}